lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -17,10 +18,20 @@ use if debug, 'Data::Dumper';
 
 sub base :Chained('/base') :PathPart('ia') :CaptureArgs(0) {
     my ( $self, $c ) = @_;
+
+    my $last_modified = $c->d->rs('InstantAnswer')->last_modified;
+    $last_modified->formatter( 'DateTime::Format::HTTP' );
+
+    $c->response->header (
+        'Last-Modified' => $last_modified;
+    );
```

Comment:
I have a feeling that doing this globally is inadequate since an update to one IA = last modified is changed for all?

@russellholt @zachthompson @MariagraziaAlastra @jdorweiler Each IA's metadata is fetched separately, right?